p4_symbolic/ir/ir.proto

```diff
@@ -93,9 +93,22 @@ message TableImplementation {
   // There is always a most general match because of P4 restrications
   // on match combination (which may be under-documented).
   // See https://github.com/p4lang/p4-spec/issues/411
-  p4_symbolic.bmv2.TableMatchType match_type = 5;
+  p4_symbolic.bmv2.TableMatchType match_type = 1;
   // Table type with respect to action selection.
-  p4_symbolic.bmv2.ActionSelectorType action_selector_type = 6;
+  p4_symbolic.bmv2.ActionSelectorType action_selector_type = 2;
+  // Concrete table entries.
+  repeated TableEntry entries = 3;
+}
+
+// Table entry parsed from P4RT/simple_switch_CLI.
+message TableEntry {
```

Comment:
Is there a reason we are not using the p4.v1.TableEntry instead of this?

Reply:
You beat me to this, I was writing a comment below about this. Please see the comment.